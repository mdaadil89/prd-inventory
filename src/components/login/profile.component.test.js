import React from 'react'
import {mount, shallow} from 'enzyme'
import Profile from './profile.component'
import toJson from 'enzyme-to-json';

describe('Profile Snapshot', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = mount(<Profile/>);
    });

    test('renders correctly', () => { 
        expect(toJson(wrapper)).toMatchSnapshot(); 
    });  
     
});

describe('Profile Page rendering of elements' , () => {
    let wrapper1;
    beforeEach(() => { 
        wrapper1 = shallow(<Profile/>)
    })
      
    it('has 5 paragraph element', () => {
        expect(wrapper1.find('li').length).toEqual(5);
    })
    
});

