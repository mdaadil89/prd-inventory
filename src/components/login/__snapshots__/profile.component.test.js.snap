// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Snapshot renders correctly 1`] = `
<Profile>
  <div>
    <br />
     
    <br />
    <br />
    <br />
    <br />
    <div
      className="container bootstrap snippet"
    >
      <div
        className="panel-body inf-content"
      >
        <div
          className="row"
        >
          <div
            className="col-md-4"
          >
            <img
              alt=""
              className="img-circle img-thumbnail isTooltip"
              data-original-title="Usuario"
              src="https://bootdey.com/img/Content/user-453533-fdadfd.png"
              style={
                Object {
                  "width": "600px",
                }
              }
              title=""
            />
            <ul
              className="list-inline ratings text-center"
              title="Ratings"
            >
              <li>
                <a
                  href="#"
                >
                  <span
                    className="glyphicon glyphicon-star"
                  />
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  <span
                    className="glyphicon glyphicon-star"
                  />
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  <span
                    className="glyphicon glyphicon-star"
                  />
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  <span
                    className="glyphicon glyphicon-star"
                  />
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  <span
                    className="glyphicon glyphicon-star"
                  />
                </a>
              </li>
            </ul>
          </div>
          <div
            className="col-md-6"
          >
            <strong>
              Information
            </strong>
            <br />
            <div
              className="table-responsive"
            >
              <table
                className="table table-condensed table-responsive table-user-information"
              >
                <tbody>
                  <tr>
                    <td>
                      <strong>
                        <span
                          className="glyphicon glyphicon-asterisk text-primary"
                        />
                        Identificacion
                      </strong>
                    </td>
                    <td
                      className="text-primary"
                    />
                  </tr>
                  <tr>
                    <td>
                      <strong>
                        <span
                          className="glyphicon glyphicon-user  text-primary"
                        />
                        Name
                      </strong>
                    </td>
                    <td
                      className="text-primary"
                    />
                  </tr>
                  <tr>
                    <td>
                      <strong>
                        <span
                          className="glyphicon glyphicon-cloud text-primary"
                        />
                        Last Name
                      </strong>
                    </td>
                    <td
                      className="text-primary"
                    />
                  </tr>
                  <tr>
                    <td>
                      <strong>
                        <span
                          className="glyphicon glyphicon-envelope text-primary"
                        />
                        Username
                      </strong>
                    </td>
                    <td
                      className="text-primary"
                    />
                  </tr>
                  <tr>
                    <td>
                      <strong>
                        <span
                          className="glyphicon glyphicon-eye-open text-primary"
                        />
                        Location
                      </strong>
                    </td>
                    <td
                      className="text-primary"
                    />
                  </tr>
                  <tr>
                    <td>
                      <strong>
                        <span
                          className="glyphicon glyphicon-phone text-primary"
                        />
                        Mobile
                      </strong>
                    </td>
                    <td
                      className="text-primary"
                    />
                  </tr>
                </tbody>
              </table>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</Profile>
`;
